### PR TITLE
Add Deposited BDV Counter to Silo

### PR DESCRIPTION
**Pull request by publiuss**

### CODE REVIEW

File: protocol/contracts/beanstalk/silo/ConvertFacet.sol [hunk 1]
```diff
@@ -197,43 +198,57 @@ contract ConvertFacet is ReentrancyGuard {
...
         // Iterate through all stems, redeposit the tokens with new BDV and
         // summate new Stalk.
         for (uint256 i; i < stems.length; ++i) {
-            uint256 bdv = amounts[i].mul(aa.bdvAdded).div(ar.tokensRemoved); // Cheaper than calling the BDV function multiple times.
+            if (i+1 == stems.length) {
+                // Ensure that a rounding error does not occur by using the 
+                // remainder BDV for the last Deposit.
+                depositBdv = newTotalBdv.sub(bdvAdded);
+            } else {
+                // bdb is a proportional amount of the total bdv.
```
Comment (pizzaman1337) [inline protocol/contracts/beanstalk/silo/ConvertFacet.sol:220]: `bdb` -> `bdv`?

Comment (Brean0) [reply protocol/contracts/beanstalk/silo/ConvertFacet.sol]: fixed: 1ea205408481c3c1d855e21d9ed1ade113e99c53

File: protocol/contracts/beanstalk/AppStorage.sol [hunk 2]
```diff
@@ -241,14 +241,16 @@ contract Storage {
     /**
      * @notice System-level Silo state; contains deposit and withdrawal data for a particular whitelisted Token.
      * @param deposited The total amount of this Token currently Deposited in the Silo.
+     * @param bdv The total bdv of this Token currently Deposited in the Silo.
```
Comment (Brean0) [inline protocol/contracts/beanstalk/AppStorage.sol:244]: should this be `depositedBdv`?

Comment (publiuss) [reply protocol/contracts/beanstalk/AppStorage.sol]: See that you fixed it here:
https://github.com/BeanstalkFarms/Beanstalk/pull/457/commits/09ecaa347562ef4916399cb95c9d66ff52ebad55
ty

**Pull request merged**